Gopkg.toml

```diff
@@ -29,7 +29,7 @@ required = [
 
 [[constraint]]
   name = "github.com/cenkalti/backoff"
-  version = "1.1.0"
+  version = "2.0.0"
```

Comment:
Rather than update the version here could we add giantswarm/backoff and use the master branch?

So if we update the library we don't need to change it here.

Reply:
I think a PE should do the grind. I do not want the SREs to hustle with this shit.

Reply:
I am fine when Roma just makes it work for him so he can move on with his stories.

Reply:
Ack. 👍 

No blocker on this. When it works lets go. We can change the dep stuff later.